tests/test_denormalized.c

```diff
@@ -0,0 +1,11 @@
+#include <float.h>
```

Comment:
Maybe I'd call the file test_cop1 or test_float. Typically a file might contain multiple related tests, so the name should be more generic.